css/style.css

```diff
@@ -850,13 +860,22 @@ h2 a {
 
 .algolia-autocomplete {
   max-width: 9em;
-  top: -0.2em;
-  margin-right: 3px;
 }
 
 .algolia-autocomplete #q {
-  display: initial
+  display: initial;
+  border-radius: 8px;
+  border: 1px solid #ddd;
+  transition: color .3s ease;
+  padding-inline: 12px;
+  outline: none;
 }
+
+.algolia-autocomplete #q:focus-visible,
+.algolia-autocomplete #q:focus {
+  border-color: #333 !important;

```

Comment:
Ge rid of important if possible. Looks like it might have been there already, but would be good go get rid of it now!